pkg/virt-operator/kubevirt_test.go

```diff
@@ -2624,6 +2624,73 @@ var _ = Describe("KubeVirt Operator", func() {
 			Expect(kvTestData.resourceChanges["daemonsets"][Patched]).To(Equal(0))           // namespace unpatched
 		})
 
+		It("should apply virt-api replica count from a CustomizeComponents patch", func() {
+			kvTestData := KubeVirtTestData{}
+			kvTestData.BeforeTest()
+			defer kvTestData.AfterTest()
+
+			expectedReplicas := 4
```

Comment:
Make this a const?

Reply:
Please also make int int32, to prevent the casting below.
```suggestion
			const expectedReplicas int32 = 4
```